internal/pkg/java/conn.go

```diff
@@ -143,5 +143,14 @@ func (pc ProcessedConn) DisconnectServerNotFound() error {
 			JSONResponse: protocol.String(msg),
 		}.Marshal()
 	}
-	return pc.WritePacket(pk)
+	if err := pc.WritePacket(pk); err != nil {
+		return err
+	}
+
+	pingPk, err := pc.ReadPacket()
+	if err != nil {
+		return err
+	}
+
+	return pc.WritePacket(pingPk)
```

Comment:
We have actually already read the ping packet at this point. It is stored in the readPks slice. So you could just write
```golang
return pc.WritePackets(pk, pc.readPks[1])
```
Packets are read here https://github.com/haveachin/infrared/blob/bee5cb005780b6d2d63a0efc00d0b85f661907d5/internal/pkg/java/conn_processer.go#L32